conda_smithy/templates/appveyor.yml.tmpl

```diff
@@ -58,4 +61,8 @@ build: off
 test_script:
     - conda.exe build {{ recipe_dir }} -m .ci_support\%CONFIG%.yaml
 deploy_script:
+  {%- if upload_on_branch %}
+    - if "%APPVEYOR_REPO_BRANCH%" == "%UPLOAD_ON_BRANCH%" (cmd: upload_package .\ .\{{ recipe_dir }} .ci_support\%CONFIG%.yaml) else (echo "Not uploading the package because %UPLOAD_ON_BRANCH% does not match %APPVEYOR_REPO_BRANCH%.")
```

Comment:
Can this be done in the CI scripts in https://github.com/conda-forge/conda-forge-ci-setup-feedstock ?

Reply:
@isuruf, could you please elaborate on that point? How can it be accomplished?

Reply:
In https://github.com/conda-forge/conda-forge-ci-setup-feedstock/blob/master/recipe/conda_forge_ci_setup/upload_or_check_non_existence.py#L92, you can check this condition.

Reply:
Sounds great, thanks for the link! Will it work equivalently for all platforms if it's implemented there?

Reply:
On second thought, we can't do that because the env variables are not sent to the docker image.